riscv/tests/riscv.rs

```diff
@@ -86,6 +86,14 @@ fn test_password() {
     verify_crate(case, vec![]);
 }
 
+#[test]
+// TODO: If this is merged, will probably break nightly tests.
```

Comment:
Please let's not break nightly tests.

Reply:
Would be nice to have categories of disabled tests. Maybe we can turn the "too slow" tests into some kind of feature?

Reply:
Without changing the test harness, there are only two ways to filter tests: the ignore flag and the name. I guess we won't be revamping how test works in this PR, so my suggestion is to just comment out this test and leave a TODO to uncomment when we support everything revm needs.